README.md

```diff
@@ -8,29 +8,41 @@ This is a brand new take on Ansible Galaxy, so it will look and feel a bit diffe
 
 Our mission is to help organizations share Ansible automation and promote a culture of collaboration around Ansible automation development. We'll be providing features that make it easy to create, discover, use and distribute Ansible automation content.
 
-To see what we're currently working on, [view the Roadmap](/ROADMAP.rst). 
+To see what we're currently working on, [view the Roadmap](/ROADMAP.rst).
 
 To learn more about Pulp, [view the Pulp project page](https://pulpproject.org/).
 
 ## Documentation
 
-Project documentation is hosted on [Read The Docs](http://galaxy-ng.readthedocs.io/).
+Project documentation is hosted on [Read The Docs](https://ansible.readthedocs.io/projects/galaxy-ng/en/latest/).
 
 ## OpenAPI Spec
 
-View the latest version of the spec at https://galaxy.ansible.com/api/v3/swagger-ui/. *(Directlink to [JSON](https://galaxy.ansible.com/api/v3/openapi.json) or [YAML](https://galaxy.ansible.com/api/v3/openapi.yaml))*
+View the latest version of the spec at <https://galaxy.ansible.com/api/v3/swagger-ui/>. *(Directlink to [JSON](https://galaxy.ansible.com/api/v3/openapi.json) or [YAML](https://galaxy.ansible.com/api/v3/openapi.yaml))*
+
+## Communication
+
+Refer to the [Communication](https://ansible.readthedocs.io/projects/galaxy-ng/en/latest/community/overview/#communication)
+section of the Contributor Guide to find out how to get in touch with us.
+
+You can also find more information in the
+[Ansible communication guide](https://docs.ansible.com/ansible/devel/community/communication.html).
 
 ## Contributing
 
 * If you're interested in jumping in and helping out, [view the contributing guide](https://github.com/ansible/galaxy_ng/wiki#contributing-to-galaxyng).
 * To setup your development environment, [view the development setup guide](https://github.com/ansible/galaxy_ng/wiki/Development-Setup).
-* Chat with us on irc.libera.chat: #ansible-galaxy
 * Found a bug or have a feature idea? Please [open an issue](https://issues.redhat.com/projects/AAH/issues).
 
+## Code of Conduct
+
+Please see the official
+[Ansible Community Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
```

Comment:
```suggestion
Please see the [Ansible Community Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
```
Maybe an extra thing? (same in the other PRs) If it sounds better for a native ear, feel free to just mark this comment as resolved